### PR TITLE
Update wiki docs and tree reopening parts of merge duty

### PR DESCRIPTION
**Pull request by mozbhearsum**

### CODE REVIEW

**Review by escapewindow — APPROVED** [review]
r+ if it's ok with relman that we open trees without asking.

File: procedures/release-duty/merge-duty/merge_duty.rst [hunk 1]
```diff
@@ -437,9 +437,8 @@ new version.
 2. ``git checkout -b nightly_version_bump_${version}``
 3. Edit FIREFOX_NIGHTLY's major version in
    https://github.com/mozilla-releng/shipit/blob/master/api/src/shipit_api/common/config.py#L48
-4. Edit the known dates at
+4. Edit the `LAST` and `NEXT` known dates (all 6 of them) at
```
Comment (MihaiTabara) [inline procedures/release-duty/merge-duty/merge_duty.rst:440]: Thanks for fixing these, seems like last time I missed this line, oups!

File: procedures/release-duty/merge-duty/merge_duty.rst [hunk 2]
```diff
@@ -299,8 +299,8 @@ Merge central to beta
 Re-opening the tree(s)
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Ask Sheriffs and RelMan to re-open trees (either ``open`` or
-``approval-only``) so that **l10n bumper can run**.
+`Reopen the mozilla-beta tree <https://treestatus.mozilla-releng.net/static/ui/treestatus/show/mozilla-beta>`__
+by restoring the previous state so that **l10n bumper can run**.
```
Comment (MihaiTabara) [inline procedures/release-duty/merge-duty/merge_duty.rst:303]: As Aki suggested yesterday, long-term solution is to have the l10n bumper be part of the behavior and submit that without `DONTBUILD` and with `CLOSE TREES`. Tracked in https://bugzilla.mozilla.org/show_bug.cgi?id=1666321. Until then, thanks for fixing the docs!

**Review by MihaiTabara — APPROVED** [review]

**Comment by mozbhearsum** [comment]
> r+ if it's ok with relman that we open trees without asking.

@jcristau, @rvandermeulen - a?

**Comment by rvandermeulen** [comment]
Fine by me.

**Pull request merged**